models/GroupUsers.ts

```diff
@@ -21,9 +21,15 @@ import { ModelCommon, ModelStaticCommon } from "./index";
 import { GroupId } from "./Group";
 import { UserId } from "./User";
 
+export const ACCESS_NONE : number = 0;
+export const ACCESS_READ : number = 1;
+export const ACCESS_ADMIN : number = 2;
+export type ACCESS_LEVEL = number;
```

Comment:
This would be a little better as an enum type, since not just any number is allowed.  You could do ```enum AccessLevel { None = 0, Read = 1, Admin = 2 }```, or you could just say that the type of `ACCESS_LEVEL` is ```ACCESS_LEVEL = ACCESS_NONE | ACCESS_READ | ACCESS_ADMIN;```